src/db/db_user.erl

```diff
@@ -0,0 +1,32 @@
+-module(db_user).
+
+-export([create/1,
+         authenticate/2,
+         get_from_countries/1]).
+
+create(#{<<"email">> := Email,
```

Comment:
This is wrong. Don't send a json map as argument to the function db function. You either pass each value as argument or you send a map with atoms.